backends/advanced/src/advanced_omi_backend/services/memory/config.py

```
@@ -314,40 +301,15 @@ def get_embedding_dims(llm_config: Dict[str, Any]) -> int:
-            input="hello world"
+        from advanced_omi_backend.model_registry import get_models_registry
+        reg = get_models_registry()
+        if reg:
+            emb_def = reg.get_default("embedding")
+            if emb_def and emb_def.embedding_dimensions:
+                return int(emb_def.embedding_dimensions)
+    except Exception:
+        memory_logger.error(
+            f"Failed to get embedding dimensions for model '{embedding_model}': {e}. "
+            "Falling back to default dimensions."
         )
```

Ruff (0.14.8)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)